### PR TITLE
Fix configmap example in values.yaml

### PR DESCRIPTION
**Pull request by kskumgk63**
Hi,
I would like to change to an example which works because I was struggling with implementing the custom google authentication. If we want to customize the logic of authentication, we just enable the example code.

### CODE REVIEW

File: charts/unleash/values.yaml [hunk 1]
```diff
@@ -24,90 +24,82 @@ autoscaling:
 # a shared secret for secure communication with a backend service.
 configMaps: {}
 #  index.js:
-#    mountPath: /index.js
+#    mountPath: /unleash/index.js
```
Comment (kskumgk63) [inline charts/unleash/values.yaml:27]: Working directory in a container is `/unleash`, so we have to put this file to `/unleash/index.js`.

**Comment by ivarconr** [comment]
Sorry for the late response :/ wish GitHub could remind me about outstanding PR. 

Would you mind doing a rebase against master? If not I will try to get it done next week.

**Comment by kskumgk63** [comment]
@ivarconr Fixed a conflict! Could you review it?

**Review by ivarconr — APPROVED** [review]
LGTM ;)

**Comment by ivarconr** [comment]
> unleash => (version: "2.1.0", path: "charts/unleash") > Chart version not ok. Needs a version bump!

Would need a version bump to 2.1.1 ;)

**Comment by kskumgk63** [comment]
@ivarconr Bumped tov2.1.1`! 8c8da45

**Pull request merged**